docs/build.sh

```diff
@@ -4,9 +4,12 @@
 
 rm -rf _build
 
-pandoc -s -t rst pact-reference.md pact-functions.md -o pact-reference.rst
+pandoc -s -t rst pact-reference.md -o pact-reference.rst
+pandoc -s -t rst pact-functions.md -o pact-functions.rst
+pandoc -s -t rst pact-properties.md -o pact-properties.rst
 
 # escape +, - headings
 perl -p0777i -e 's/^(\+|\-)\n~/\\\1\n~~/gm' pact-reference.rst
+perl -p0777i -e 's/^(\+|\-)\n~/\\\1\n~~/gm' pact-functions.rst
```

Comment:
should this also add `pact-properties.rst`?

Reply:
i didn't b/c this is for mangling function names. Probably should do pact-reference even but since it was already i left it in. In fact, DocGen.hs should pre-mangle so we can at least see what the refs look like and use those in the rest of the docs.